src/misc/constants.ts

```diff
@@ -35,3 +35,22 @@ export const SOUND_STREAM = 4;
 export const SOUND_UPDATE = 16;
 
 export const MAX_LAYER : number = 32;
+
+///All relevant planes used by Yogstation categorized into an enum of their values
+export const enum Planes{

```

Comment:
Shouldn't this const enum be capitalized?

Reply:
no